cristin-project/src/test/java/no/unit/nva/cristin/projects/RandomProjectDataGenerator.java

```diff
@@ -120,6 +121,7 @@ public static NvaProject randomNvaProject() {
                                           .withHealthProjectData(randomHealthProjectData())
                                           .withApprovals(randomApprovals())
                                           .withExemptFromPublicDisclosure(randomBoolean())
+                                          .withWebPage(randomUri())
```

Comment:
How about adding a randomUri(withEndpoint) to give us a clue to where a randomUri should have pointed to?

Reply:
this webpage can be anything, it does not have a common domain